azure-pipelines.yml

```diff
@@ -47,10 +47,25 @@ jobs:
         testRunTitle: 'Publish test results for Python $(python.version) $(manylinux_flag)'
 
     - task: PublishCodeCoverageResults@1
-      inputs: 
+      inputs:
         codeCoverageTool: Cobertura
         summaryFileLocation: '$(System.DefaultWorkingDirectory)/*coverage.xml'
 
+    - script: |
+        python -m pip install twine
+        make dist
+      displayName: 'Create packages'
+
+    - task: TwineAuthenticate@0
+      inputs:
+        artifactFeeds: TODOproj/TODOfeed
+        externalFeeds: pypi
```

Comment:
Maybe pypi will be available by default without listing it here? If listed here, I think that has to be set up somewhere in the azure pipelines web interface (maybe)?